project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 0.13.15
+sbt.version = 0.13.16
```

Comment:
If you use any new method/field/class of sbt 0.13.16, yes. But don't dwell too much of it. sbt-scalajs 0.6.20 will force sbt 0.13.16 on its users anyway.